scripts/update-dependencies/Program.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.VersionTools;
+using Microsoft.DotNet.VersionTools.Automation;
+using Microsoft.DotNet.VersionTools.Dependencies;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.Dotnet.Scripts
+{
+    public static class Program
+    {
+        private static readonly Config config = Config.Instance;
+
+        public static void Main(string[] args)
+        {
+            Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
+
+            if (!ParseArgs(args))
+                return;
+
+            CreatePullRequest();
+        }
+
+        private static bool ParseArgs(string[] args)
```

Comment:
Can this use CommandLineUtils (the one from NuGet.org would suffice)? Or is the input format dictated by something outside our control?